analyzer/analyzer.go

```diff
@@ -118,37 +112,19 @@ func RequiredFilenames() []string {
 
 // TODO: Remove opts as they're no longer needed
 func (ac Config) Analyze(ctx context.Context, imageName string, opts ...types.DockerOption) (fileMap extractor.FileMap, err error) {
-	r, err := ac.Extractor.SaveLocalImage(ctx, imageName)
+	transports := []string{"docker-daemon:", "docker://"}
```

Comment:
This might be outside of the scope of this PR, it would be great to allow non-docker stored images here as well, so trivy could operate on images built with buildah without doing a docker conversion.
https://github.com/containers/image/blob/master/docs/containers-transports.5.md#containers-storagestorage-specifierimage-iddocker-referenceimage-id

Reply:
Thank you for the idea! Actually, one of the reasons for using containers/image is exactly that. I'd like to support OCI and `containers-storage` in the future.